python/lsst/analysis/ap/nb_utils.py

```diff
@@ -70,3 +78,203 @@ def make_simbad_link(ra, dec, radius_arcsec=3.0):
         print(f"No matched sources within {radius_arcsec} arcseconds.")
 
         return None
+
+
+def compareDB(repo1, repo2, coll1, coll2, query1, query2,
```

Comment:
Maybe call it `compare_apdb_sources`? Or something else more descriptive (and also snake_case, which this package is mostly following).

Also, this feels like something that would be useful as a commandline tool (like the cutout plotter). Is there anything notebook-specific about it?

Reply:
No, nothing notebook-specific at all modulo actually displaying the cutouts. There's already an option to turn it off though, so it could certainly be used in other places in that mode.

Reply:
Changed to `compare_sources`.